.github/actions/e2e-deploy-vald/action.yaml

```diff
@@ -108,12 +85,13 @@ runs:
         HELM_EXTRA_OPTIONS: ${{ inputs.helm_extra_options }}
         WAIT_FOR_SELECTOR: ${{ inputs.wait_for_selector }}
         WAIT_FOR_TIMEOUT: ${{ inputs.wait_for_timeout }}
-    - name: deploy vald from local charts
+
+    - name: Deploy vald from local charts
       shell: bash
       id: deploy_vald_local
       if: ${{ inputs.use_local_charts == 'true' }}
       run: |
-        make k8s/vald/deploy HELM_VALUES=${VALUES} HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS}"
+        make k8s/vald/deploy VERSION=nightly HELM_VALUES=${VALUES} HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS}"
```

Comment:
Maybe it's better to let `"Determine Docker image tag"` workflow to do the fall back to `nightly`?